app/views/helpers/tags.php

```diff
@@ -102,14 +102,14 @@ public function displayTagsModule($tagsArray, $sentenceId = null)
      * @param null $date
      */
     public function displayTag(
-        $tagName, $tagId, $sentenceId, $userId, $username = null, $date = null
+        $tagName, $tagId, $sentenceId, $userId, $sentenceLang = null, $username = null, $date = null
```

Comment:
When you modify the signature of a function to add a new parameter, it is generally better to add the new parameter at the very end, especially if it is an optional parameter. This way there is less risk to break something. If you choose to add it somewhere in the middle, you have to make sure to fix all the usages of the function.

In our case, you will need to fix the params in `app/views/tags/add_tag_post.ctp`.
Cf. https://github.com/Tatoeba/tatoeba2/search?utf8=%E2%9C%93&q=displayTag